lib/scsi_defs.h

```diff
@@ -82,6 +82,32 @@ namespace scsi_defs {
     uint8_t vendor_specific : 2;
   };
 
+  // SCSI Reference Manual Table 202 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  struct TestUnitReadyCommand {
+    OpCode op_code = OpCode::kTestUnitReady;
```

Comment:
Should we split these out? [Discussion elsewhere](https://github.com/googleinterns/scsi2nvme/pull/30#discussion_r432001349).